ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -154,24 +154,29 @@ def create_catalog_source_yaml(self):
         except Exception as e:
             logger.info(e)
 
-    def install_ocs_operator(self):
+    def install_operator(self, operator_name):
         """
-        Install OCS Opeartor
+        Install Opeartor
 
         """
         self.navigate_operatorhub_page()
 
-        logger.info("Search OCS Operator")
-        self.do_send_keys(
-            self.dep_loc["search_operators"], text="OpenShift Container Storage"
-        )
+        logger.info(f"Search {operator_name} Operator")
+        self.do_send_keys(self.dep_loc["search_operators"], text=operator_name)
 
-        logger.info("Choose OCS Version")
-        self.do_click(self.dep_loc["choose_ocs_version"])
+        logger.info(f"Choose {operator_name} Version")
+        if operator_name == "OpenShift Container Storage":
+            self.do_click(self.dep_loc["choose_ocs_version"])
```

Comment:
consider passing to the function the actual clickable name of the operator, such as `choose_ocs_version`. This way you will not have to put these conditions here.

You can also save `choose_ocs_version` as a constant, something like `UI_OCS_OPERATOR_CLICK`